fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/EventKeeperTranslator.java

```diff
@@ -144,7 +146,7 @@ public boolean equals(final Object o) {
 
         @Override
         public int hashCode() {
-            return Objects.hash(name);
+            return hashCode;
```

Comment:
I actually suspect `name.hashCode` would be sufficient. `String.hashCode` isn't expensive: it has a cache. It's `Objects.hashCode`, which is unnecessary with a single (non-null) argument, that's taking the time.

Reply:
agreed.